tests/integration/security/citadel/namespace_targeting_test.go

```diff
@@ -0,0 +1,93 @@
+package citadel
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+
+	v1 "k8s.io/api/core/v1"
+	mv1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	caOverrideLabel    = "ca.istio.io/override"
```

Comment:
All the  constants can be referenced from the same implementation pkg without re-hardcoded.

Reply:
s/All/label names/g

Reply:
done